keras_cv/datasets/pascal_voc/segmentation.py

```diff
@@ -300,6 +362,25 @@ def _load_images(example):
     return example
 
 
+def _load_sbd_images(image_file_path, seg_cls_file_path, seg_obj_file_path):
+    image = tf.io.read_file(image_file_path)
+    image = tf.image.decode_jpeg(image)
+
+    segmentation_class_mask = scipy.io.loadmat(seg_cls_file_path)
```

Comment:
Can we do this simply without a scipy dependency?

Reply:
`tfds.core.lazy_imports.scipy.io.loadmat`

Reply:
Done. I was under the impression that tfds will still try to import scipy.io? What makes one succeed and the other fail in cloudbuild?